pkg/extensions/config/events/config.go

```diff
@@ -0,0 +1,27 @@
+package events
+
+// Config holds configuration for the events extension.
+type Config struct {
+	Enable *bool
+	NATS   *NATSConfig
```

Comment:
Guessing you started this work before the final discussion in the issue. Since you plan to use CloudEvents, maybe plug that in here somehome. There could be multiple event sinks folks that folks may configure at the same time?

Reply:
> Guessing you started this work before the final discussion in the issue. 
Yes :)

> Since you plan to use CloudEvents, maybe plug that in here somehome. 

I will look into that, ideally we could leverage a common protocol.

> There could be multiple event sinks folks that folks may configure at the same time?

I think we should initially support sending events to a single sink to avoid complexity.